set.mm

```diff
@@ -162551,6 +162616,144 @@ the complex numbers (sorting by real part first, then by imaginary part as
       CDEUDUEUF $.
   $}
 
+  $( The square of the real component of the square root of a complex number is
+     a non-negative real number.  (Contributed by RP, 11-May-2024.) $)
```

Comment:
wrong comment.

Reply:
Re[Sqrt[z]]^2 = (Abs[z] + Re[z])/2 holds for all z.